Apps/Src/BlinkLights.c

```diff
@@ -0,0 +1,10 @@
+#include "Tasks.h"
+#include "Lights.h"
+
+/**
+* @brief This thread controls the blinking of the hazard and indicator lights at the regulation specified frequency. 
+**/
+
+void Task_BlinkLight(void* p_arg){
```

Comment:
Why did you add this?

Reply:
oops that was meant to be in the new blinklights issue i'm working on